mockall/Cargo.toml

```diff
@@ -53,7 +52,7 @@ mockall_double = { version = "^0.3.1", path = "../mockall_double" }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
-tracing = "0.1.32"
+tracing = "0.1.35"
```

Comment:
Why was it necessary to change this?

Reply:
A bug in `tracing` (actually its dependency `tracing-core`) below 0.1.35 when lazy_static version is too low.
It was raising an error in the CI when testing with `cargo update -Zminimal-versions`.